index.html

```diff
@@ -2,230 +2,441 @@
-          <a href="#"><img src="images//twitter.png" /></a>
-        </li>
-        <li>
-          <a href="#"><img src="images//Linkedin.png" /></a>
-        </li>
-        <li>
-          <a href="#"><img src="images//m.png" /></a>
-        </li>
-        <li>
-          <a href="#"><img src="images//github.png" /></a>
-        </li>
-        <li>
-          <a href="#"><img src="images//fingers.png" /></a>
-        </li>
-      </ul>
-      <button>Get my resume</button>
-
-      <ul class="skills">
-        <li class="skill">
-          <span class="label"
-            >Languages
-            <img src="images//Icon-Chevron-Down.png" alt="Icon Chevron Down "
-          /></span>
-          <ul class="items">
-            <li>
-              <img src="images/Ellipse_1.png" alt="javascript" /> JavaScript
+    <section class="projects-section-desktop"></section>
+    
+    <section class="about">
+      <div class="about-mobile" id="about__me">
+        <h2 class="headline-text">
+          About<br />
+          Myself
+        </h2>
+          <p class="headline-paragraph">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+        <div class="heading-social">
+          <h2 class="icons-header">LET'S CONNECT</h2>
+          <div>
+            <ul class="icons" id="social_media">
+              <li>
+                <a href="#"
+                  ><img src="images/twitter.png" alt="twitter"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="images/Linkedin.png" alt="Linkedin"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="images/medium.png" alt="Medium"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="images/github.png" alt="Github"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="images/angellist.png" alt="Angellist"
+                /></a>
+              </li>
+            </ul>
+          </div>
+          
+        </div>
+        </div>
+      <div id="about" class="about-lang">
+        <div>
+          <ul class="lang">
+            <li><h4>Languages</h4></li>
+            <li class="arrows">
+              <img src="images/arrow-down.png" alt="arrow.png" />
+            </li>
+          </ul>
+        </div>
+        <ul class="icons-lang">
+          <li class="lang-icons"><img src="images/js.png" alt="js" />Javascript</li>
+          <li class="lang-icons"><img src="images/html.png" alt="html" />HTML</li>
+          <li class="lang-icons"><img src="images/css.png" alt="Ellipse" />CSS</li>
+        </ul>
```

Comment:
Good job here 💯 but make sure to adjust the position of the text and the image,  the text is supposed to be below, this is what yours looks like 👇🏽 

![Screenshot 2023-03-31 at 00 59 06](https://user-images.githubusercontent.com/45739963/229026813-c2e763d6-7634-4375-9b4b-a7cfa62844a2.png)

And this is what the Figma looks like 👇🏽 

![Screenshot 2023-03-31 at 01 00 45](https://user-images.githubusercontent.com/45739963/229027058-3ea6c41f-c062-44a4-8b06-c4602b9d7884.png)

Reply:
Noted and thank you for your kind suggestion.